superflore/generators/bitbake/ros_meta.py

```diff
@@ -29,16 +29,36 @@ def __init__(
             info('Creating new branch {0}...'.format(self.branch_name))
             self.repo.create_branch(self.branch_name)
 
-    def clean_ros_recipe_dirs(self, distro=None):
-        if distro:
-            info(
-                'Cleaning up meta-ros{}-{}/generated-recipes directory...'
-                .format(yoctoRecipe._get_ros_version(distro), distro))
-            self.repo.git.rm('-rf', 'meta-ros{}-{}/generated-recipes'.format(
-                yoctoRecipe._get_ros_version(distro), distro))
-        else:
-            info('Cleaning up generated-recipes directories...')
-            self.repo.git.rm('-rf', 'generated-recipes')
+    def clean_ros_recipe_dirs(self, distro):
+        info(
+            'Cleaning up:\n'
+            'meta-ros{0}-{1}/generated-recipes\n'
+            'meta-ros{0}-{1}/conf/ros-distro/include/{1}/generated\n'
+            'meta-ros{0}-{1}/files/{1}/generated/'
+            '{{newer-platform-components.list,rosdep-resolve.yaml,'
+            'superflore-change-summary.txt}}\n'
```

Comment:
As of https://github.com/ros-infrastructure/superflore/pull/273, `superflore-change-summary.txt` is no longer generated.

Reply:
But it still be removed so that we don't need to remove it manually when regenerating the recipes, right?

Reply:
Yes. Add a comment explaining this.

Reply:
Which PR is going to be merged first, because with this PR the superflore-change-summary.txt is still being generated.

Reply:
#273 will be merged first -- I've added to note to the PR description reminding us of this.

Reply:
#273 has been merged.